_posts/newsletters/2024-01-24-newsletter.md

```diff
@@ -0,0 +1,382 @@
+
+For US-RSE members, joining our meeting is a chance to get familiar with the kinds of issues user experience designers and researchers grapple with.  
+
+The UXWG strives for regular, productive meetings that accommodate multiple schedules. We meet on the 3rd Tues of odd months at 2 PM ET/1 PM CST/11 AM PT/12 PM MT, and, on even months, the 3rd Wed at 4:30 PM ET/3:30 PM CST/1:30 PM PT/12 PM MT.
+
+Next meeting: Feb 21, 4:30 PM Eastern  
+[https://utah.zoom.us/j/97840166678?pwd=Yi9kZTZ2eGVib3pNeWduSjNqVnNvUT09](https://utah.zoom.us/j/97840166678?pwd=Yi9kZTZ2eGVib3pNeWduSjNqVnNvUT09)  
+Meeting ID: 978 4016 6678   
+Passcode: 208039
+
+Join us on Slack in the [#wg-ux](https://app.slack.com/client/T8ZT4PJSW/C0664AMJBKQ) channel.
+
+## Code Review Working Group
+The goal of this working group is to build a community of RSEs who are interested in code review. The working group will provide a forum to gather resources related to code review, discuss the use of code review in research software project, and seek help in conducting code reviews.
+
+To get involved, visit the [#wg-code-review](https://app.slack.com/client/T8ZT4PJSW/C063DTDEP0A) channel on the US-RSE slack.
+
+<a name="community-calls"></a>
+## Community Calls
+ 
+### February Community Call 
+
+**Call for Presenters**
+
+If you are interested in presenting, please get in touch with Julia Damerow and Abbey Roelofs in the [#communitycalls](https://app.slack.com/client/T8ZT4PJSW/CLBDQMJH5) channel or by messaging them directly.
+
+
+
+### January Community Call (Past)
+**The January Community Call was on the topic of Python Frameworks.**
+
+Speakers presented on best practices and experiences they have had with Python Frameworks.
+
+
+<a name="community-manager"></a>
+## Community Manager Office Hours
+One weekday and one weekend time slot will be set aside each month for US-RSE members to chat with Sophia about membership matters. Members may also provide updates on how they are applying technical tools and other resources they have learned about from the US-RSE community. Additional information and relevant resources can be found on the US-RSE Slack in the [#community-manager](https://app.slack.com/client/T8ZT4PJSW/C06E5P599JN) channel.
+
+
+<a name="events"></a>
+# **Interesting Events and Opportunities**
+
+## Community and Travel Funds Program
+
+Part of the Alfred P. Sloan Foundation grant for US-RSE has been delegated for the [Community and Travel Funds program](https://us-rse.org/grants-and-awards/). Members of US-RSE can apply for funds for community or individual purposes for event costs, get-togethers, travel funding, and more. Apply today!
+
+
+## Survey to understand how members of the research software engineering community consider cyber security when writing software
```

Comment:
Minor nit: this looks really long when rendered as a section heading.  Maybe replace with something like "Call for participants in cybersecurity survey"?

Reply:
I will use your suggestion.

Reply:
I will change it to this: Survey to understand how RSEs consider cybersecurity when writing software